src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java

```diff
@@ -472,7 +469,6 @@ public int compare(Pair<Coordinate, Double> coordinateDoublePair, Pair<Coordinat
 
         LineSegment normalizedLineSegment = new LineSegment(p1Transformed, p2Transformed);
 
-
         //now normal state is reached and Derivative can be applied for MaxConstant Search
```

Comment:
There are multiple inline comments, which I've not found. Please replace and write self-explaining methods. And If you want to explain it nevertheless (which you should) add javadoc to the method